src/host/VtInputThread.cpp

```diff
@@ -57,79 +43,67 @@ VtInputThread::VtInputThread(_In_ wil::unique_hfile hPipe,
 // - The return value of the underlying instance's _InputThread
 DWORD WINAPI VtInputThread::StaticVtInputThreadProc(_In_ LPVOID lpParameter)
 {
-    const auto pInstance = reinterpret_cast<VtInputThread*>(lpParameter);
+    const auto pInstance = static_cast<VtInputThread*>(lpParameter);

```

Comment:
The diff in this file is a lot smaller with suppressed whitespace changes: https://github.com/microsoft/terminal/pull/17574/files?w=1